assets/js/add-task.js

```diff
@@ -0,0 +1,78 @@
+window.addEventListener("load", () => {
+  const form = document.querySelector("#create-task");
+  const inputTask = document.querySelector("#input-task");
+  const taskList = document.querySelector("#tasks");
+  const createBtn = document.querySelector("#create");
+  const cancel = document.querySelector("#cancel");
+
+  createBtn.addEventListener("click", () => {
+    if (form.className == "hide") {
+      form.classList.remove("hide");
+    }
+    inputTask.focus();
+  });
+
+  cancel.addEventListener("click", () => {
+    form.classList.add("hide");
+    inputTask.value = "";
+  });
+
+  form.addEventListener("submit", (event) => {
+    event.preventDefault();
+
+    if (!inputTask.value) {
+      return;
+    }
+
+    const task = document.createElement("div");
+    task.classList.add("task");
+
+    const task_content = document.createElement("div");
+    task_content.classList.add("content");
+
+    const currentDate = new Date();
+    const options = { day: "2-digit", month: "2-digit", year: "2-digit" };
+    const formattedDate = currentDate
+      .toLocaleDateString("en-UK", options)
+      .replace(/\//g, ".");
+
+    const input_element = document.createElement("input");
+    input_element.classList.add("text");
+    input_element.type = "text";
+    input_element.value = inputTask.value;
+    input_element.setAttribute("readonly", "readonly");
+
+    task_content.appendChild(input_element);
+
+    const tools = document.createElement("div");
+    input_element.classList.add("actions");
+
+    const date = document.createElement("p");
+    date.innerHTML = "Created At: " + formattedDate;
+    const done = document.createElement("button");
+    done.classList.add("done");
+    done.innerHTML = `<img src="./assets/images/check.png" alt="Done button" id="done" />`;
```

Comment:
Can we store our stings inside another file as constants and use that instead? Like here instead of writing hard codded value for the alt, we may use a constant value for that.
Also, Instead of static paths can we use relative paths?

Reply:
I have used fontawesome in the updated code. So, there is no "alt" attribute now. So should I create file to store constants now or should I create it when necessary?

Reply:
When it is necessary.